tests/mypy/test_fields.yml

```diff
@@ -44,8 +44,8 @@
     main:22: note: Revealed type is 'builtins.str'
     main:23: note: Revealed type is 'builtins.str'
     main:24: note: Revealed type is 'builtins.str'
-    main:25: note: Revealed type is 'def (self: main.Example, info: Any) -> builtins.str'
-    main:26: note: Revealed type is 'def (self: main.Example, info: Any) -> builtins.str'
+    main:25: note: Revealed type is 'strawberry.field.StrawberryField'
```

Comment:
Should _all_ the fields have revealed types as `StrawberryField`? I anticipate being able to make StrawberryField generic so I think it could be
```
main:21: note: Revealed type is 'strawberry.field.StrawberryField[str]'
```
for each.

Reply:
that's a good question, that's only for the class? The instance types would still be 'builtins.str', right?